data/mods/joltemons/random-teams.ts

```diff
@@ -0,0 +1,1870 @@
+/* eslint max-len: ["error", 240] */
+
+
+import {Dex, toID} from '../../../sim/dex';
+import {PRNG, PRNGSeed} from '../../../sim/prng';
+
+export interface TeamData {
+	typeCount: {[k: string]: number};
+	typeComboCount: {[k: string]: number};
+	baseFormes: {[k: string]: number};
+	megaCount: number;
+	zCount?: number;
+	has: {[k: string]: number};
+	forceResult: boolean;
+	weaknesses: {[k: string]: number};
+	resistances: {[k: string]: number};
+	weather?: string;
+	eeveeLimCount?: number;
+}
+
+export class RandomTeams {
```

Comment:
This should inherit from `RandomTeams` rather than declaring its own standalone class; did you just copy/paste this from somewhere?

Reply:
It seems like an older version of the file, too.

Reply:
i've copied the code from [https://github.com/scoopapa/DH/tree/master/data/mods/joltemons](the version of joltemons on DragonHeaven) so you'd have to ask @Yoshiblaze

Reply:
i've made the changes that were more straightforward, editing the random-teams file sounds too daunting for me

Reply:
This isn't how randomized team generation for mods _should_ be implemented; even if side servers are willing to duplicate lots of code, that shouldn't happen in the main repository. Since you're too daunted, I'll implement it myself and add it to the PR.